tools/release.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
```

Comment:
I think this file is not needed, since it already exists in https://github.com/celo-org/celo-bls-go/blob/master/scripts/release.sh